src/components/evaluations/info/FeedbackPm.vue

```diff
@@ -23,9 +24,10 @@
                 <samp class="small">Your test results can be found here: http://bn.mappersguild.com/testresults</samp><br><br>
                 <samp class="small">Additional feedback regarding why you were rejected and what you could potentially improve in your next application:</samp><br><br>
                 <samp><pre class="small">[notice]{{ feedback }}[/notice]</pre></samp>
-                <samp class="small">For further feedback or to appeal this decision, contact any of these users:
+                <samp class="small">If you have questions or wish a more in-depth feedback contact:, contact any of these users:
```

Comment:
```suggestion
                <samp class="small">If you have questions or wish more in-depth feedback, contact any of these users:
```
Grammar (feedback is a mass noun, so can't do "a feedback") + duplicated contact